documentation/crds_users_guide/source/command_line_tools.rst

```diff
@@ -211,7 +211,7 @@ New Context
 
 crds.bestrefs always computes best references with respect to a context which
 can be explicitly specified with the `--new-context` parameter.  If `--new-context`
-is not specified, the default latest context is determined by consulting
+is not specified, the default build context is determined by consulting
```

Comment:
This is only the case for JWST. This needs to be clarified with something like:  "{...} is not specified, the way the context is determined depends on the observatory.  For Roman and HST, the default latest context is determined by consulting the CRDS server or looking in the local cache. For JWST, the default build context is determined by consulting the CRDS server to find the appropriate context for the calibration software version you have locally installed."